<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="wc3_table">
    <pose>0 0 0.4572 0 0</pose>
    <static>0</static>
    <link name="base_link">
      <visual name="visual_model">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
            <mesh>
              <uri>model://wc3_table/meshes/wc3_table_visual.stl</uri>
            </mesh>
        </geometry>
        <material>
          <ambient>0.44 0.33 0.23 1</ambient>
          <diffuse>0.44 0.33 0.23 1</diffuse>
          <specular>0 0 0 0</specular>
          <emissive>0 0 0 1</emissive>
        </material>
      </visual>
      <collision name="collision_model">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
            <mesh>
              <uri>model://wc3_table/meshes/wc3_table_collision.stl</uri>
            </mesh>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>100</mu>
              <mu2>50</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <inertial>
        <mass>50.0</mass>
        <pose>-0.0062 -0.0683 0.0536 0.000 0.000 0.000</pose>
        <inertia>
          <ixx>5.8162</ixx>
          <ixy>0.0000</ixy>
          <ixz>0.0000</ixz>
          <iyy>6.8527</iyy>
          <iyz>0.0000</iyz>
          <izz>6.0355</izz>
        </inertia>
      </inertial>
    </link>
  </model>

</sdf>
